src/ddmd/dclass.d

```diff
@@ -179,6 +179,28 @@ struct ClassFlags
     alias hasDtor = Enum.hasDtor;
 }
 
+/**
+ * The ClassKind enum is used in ClassDeclaration AST nodes
+ * to specify the linkage type of the class or if it is an
+ * anonymous class. If the class is anonymous it is also
+ * considered to be a D class.
+ */
+enum ClassKind : ubyte
+{
+    /// the class is a d(efault) class
+    d,
+    /// the class is a C++ interface
+    cpp,
+    /// the class is an Objective-C class/interface
+    objc,
+    /// the class is anonymous
+    anonymous,
+}
+
+alias CKcpp = ClassKind.cpp;
+alias CKobjc = ClassKind.objc;
+alias CKanonymous = ClassKind.anonymous;
```

Comment:
I suggest to remove these aliases and always use the fully name of the enum members, it's more D like.

Reply:
Maybe we should do some refactoring and get rid of more aliases?

Reply:
I agree, but let's start by not making the adding more 😃.